### PR TITLE
Setup React TS & Build Tools for popup.html

### PR DESCRIPTION
**Pull request by oal55**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1497 @@
+{
```
Comment (oal55) [inline package-lock.json:1]: Sorry about this! We can now use react though :tada

File: content.js [hunk 2]
```diff
@@ -1,36 +1,40 @@
...
-  popupDiv.innerHTML = `<p>Selected Word: ${selectedText}</p>`;
-
-  // Append the popup to the body
-  document.body.appendChild(popupDiv);
-
-  return popupDiv;
-}
+import { add } from "./lib/utils";
+
+console.log('add', add);
```
Comment (oal55) [inline content.js]: I can undo these, was experimenting with module syntax 👀

File: package-lock.json (hunk 1 above)
Comment (oal55) [reply package-lock.json:1]: 🎉  *

**Review by irmk323 — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (irmk323) [reply package-lock.json:1]: 💯

**Pull request merged**